pages/advanced-algorithms/available-algorithms/merge.mdx

```diff
@@ -0,0 +1,176 @@
+
+Finally, let's age every 20-year-old to 25.
+
+```cypher
+CALL merge.node([],{age: 20},{},{age: 25}) YIELD node RETURN node;
+```
+
+
+```plaintext
++---------------------------------------------------------------------------------------------------------------------------------+
+| node                                                                                                                            |
++---------------------------------------------------------------------------------------------------------------------------------+
+| {"id": 0,"labels": ["Human"],"properties": {"age": 25,"galaxy": "Milky way","name": "Mark","weapon": "sword"},"type": "node"}   |
++---------------------------------------------------------------------------------------------------------------------------------+
+| {"id": 2,"labels": ["Alien"],"properties": {"age": 25,"galaxy": "Milky way","name": "Gorlock"},"type": "node"}                  |
++---------------------------------------------------------------------------------------------------------------------------------+
+| {"id": 33,"labels": ["Zumbi"],"properties": {"age": 25,"galaxy": "Andromeda"},"type": "node"}                                   |
++---------------------------------------------------------------------------------------------------------------------------------+
+```
+
+### `relationship()`
+
+Provides a more flexible way of merging relationships than Cypher’s `MERGE` clause, by
+allowing labels and properties to be extracted as results of other procedures
+and sent as a list or map.
+
+{<h4> Input: </h4>}
+
+- `startNode: Node` ➡ Start node of the relationship.
+- `relationshipType: string` ➡ Relationship type to be matched or created.
+- `identProps: Map` ➡ A map with key/value pairs used for identifying which relationships will be merged.
+- `createProps: Map` ➡ A map with key/value pairs which represent properties of the relationship, when the procedure creates a new relationship.
+Equivalent to using `ON CREATE SET` for properties when using `MERGE`.
+- `endNode: Node` ➡ End node of the relationship.
+- `mergeProps: Map` ➡ A map with key/value pairs which represent properties of the relationship, when the procedure merges onto a relationship.
+Equivalent to using `ON MATCH SET` for properties when using `MERGE`.
+
+{<h4> Output: </h4>}
+
+- `rel: Relationship` ➡ The relationship which was created or merged in the procedure.
+
+{<h4> Usage: </h4>}
+Consider a graph with one `Human`, two `Dog` nodes. Both dogs love the human but one loves him unconditionally.
+
+```cypher
+CREATE (d:Dog {name: 'Johnny'})-[l:Loves {unconditionally: true}]->(h:Human) CREATE (c:Dog {name: 'Floki'});
+```
+Let's use `merge.relationship` to add a property called `trueLove` to every `Loves` relationship between a human and a dog that has the property unconditional set to true. Otherwise create a new `Loves` relationship with the property unconditionally set to false.
```

Comment:
```suggestion

Use `merge.relationship` procedure to add a property called `trueLove` to every `Loves` relationship between a human and a dog that has the property unconditional set to `true`. Otherwise, create a new `Loves` relationship with the property unconditionally set to `false`.
```